docs/oathkeeper/grpc-middleware.md

```diff
@@ -0,0 +1,33 @@
+---
+id: grpc-middleware
+title: gRPC middleware
+---
+
+In addition to exposing Ory Oathkeeper as a HTTP proxy, you can directly embed it into your Go application as a gRPC middleware.
```

Comment:
Can you please also document the new gRPC matcher in the configuration?

Reply:
done.